docs/getting-started/github-apps.rst

```diff
@@ -45,47 +44,117 @@ sync with the linked GitHub repository.
+testParameterFiles (optional)
+    An array of absolute paths to test parameter files in the Git repository.
+
+Ex. .dockstore.yml with a single workflow
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test/dockstore.cwl.json
+
+The above ``.dockstore.yml`` is for a single workflow. Note that the name is not present since it is optional.
+
+Ex. .dockstore.yml with multiple workflows
+
+.. important:: The **name** field is an optional field used when a repository has multiple workflows in it that a user wants to register
+    as separate entries on Dockstore. Each entry within a ``.dockstore.yml`` file corresponds to a unique entry on Dockstore.
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - name: globalAligner
+        subclass: CWL
+        primaryDescriptorPath: /runGlobalAligner.cwl
+        testParameterFiles:
+            - /test/globalAligner.cwl.json
+      - name: localAligner
+        subclass: CWL
+        primaryDescriptorPath: /runLocalAligner.cwl
+        testParameterFiles:
+            - /test/localAligner.cwl.json
+
+A common pattern seen on Dockstore is GitHub repositories that store many workflows. The above ``.dockstore.yml``
+has two entries for workflows. Notice that each entry uses a different name. Names are required if you want 
+multiple workflows registered on Dockstore from a single GitHub repository. The names must be unique between
+entries of the `workflows` array. For each unique name present, an entry will be created on Dockstore.
 
 Service YML File
 +++++++++++++++++
 TODO
 
+Migrating Existing Workflows to use GitHub Apps
+-------------------------------------------------
+If you have already registered GitHub workflows on Dockstore using our old method which required you to refresh, you are still able to use GitHub Apps!
+The migration process is fairly straightforward. First, make sure to install our GitHub App onto the repository for the workflow that you want to migrate.
+Next, create a ``/.dockstore.yml`` file and push it to a branch on your GitHub repo, making sure that the name field matches the existing workflowname.
+If your workflow did not have a workflowname set, simply exclude the field. It is very important that these names match, or else Dockstore won't know which
+workflow to associate the ``/.dockstore.yml`` with.
+
+The pushing of a branch with a valid ``/.dockstore.yml`` should trigger Dockstore to convert your workflow into a GitHub App workflow! New versions will be added/updated/deleted
+as if the workflow was originally added using GitHub Apps.  Existing branches will persist, and you can even individually refresh them on the versions tab of the workflow.
+You can convert these existing workflows to use GitHub Apps by adding a valid ``/.dockstore.yml`` to the corresponding branches on GitHub.
```

Comment:
I'd specify "root directory of the corresponding branches..." just to be paranoid.